readme.md

```diff
@@ -101,23 +100,8 @@ Your `package.json` will then look like this:
 }
 ```
 
-Any arguments passed after `--init` are added as config to `package.json`.
-
-#### Manual installation
-
-You can also install AVA directly:
-
-```console
-$ npm install --save-dev ava@next
-```
-
-Alternatively using Yarn:
-
-```console
-$ yarn add --dev ava@next
-```
-
-You'll have to configure the `test` script in your `package.json` to use `ava` (see above).
+NOTE: Running `npx` requires `npm` version `0.5.2` or greater. Otherwise, you'll have to 
```

Comment:
It's `5.2.0`, not `0.5.2` 😉 Perhaps add a link to https://github.com/npm/npm/releases/tag/v5.2.0?